### ISSUE TITLE
Is there any available, ready-to-use model/nets?

### ISSUE THREAD

**Issue by bontorhumala** [JackKelly/neuralnilm#3]
Hi Kelly, thanks for sharing your work. I am trying to run this code, but encounter the same problem as https://github.com/JackKelly/neuralnilm/issues/2

I understand that you are not supporting this repo anymore. However, I am just curious, do you have trained nets that we can just use for your test set? Because I see each experiment code (e575, e576, etc) and I guess all of them require nets for each appliance to be created/trained from scratch. And I dont see trained nets inside neuralnilm subdirectory.

Thanks again!

**Comment by JackKelly** [comment]
Hi,

Thanks for your interest.  When I'm next in the office, I'll check to see if I have any trained neural nets that I can provide.  No promises, though!

Have a good weekend,
Thanks,
Jack

**Comment by JackKelly** [comment]
I've uploaded a few nets to my FTP server:

      host: achillea.doc.ic.ac.uk
      port: 55555
      username: anonymous
      password: <your email address>

**Comment by gohingsiang** [comment]
Hi, may i have the pretrained model as well? Thanks in advance.

**Comment by JackKelly** [comment]
thank you for your interest.  Now that I have left Imperial, I'm afraid that pre-trained models are no longer available.  Sorry.

I'd highly recommend using a modern DeepLearning framework like TensorFlow for NILM work now-a-days.

**Comment by gohingsiang** [comment]
Thanks for the reply, maybe i will just do it step by step. By the way, your advice was most appreciate.

**Comment by maechler** [comment]
For anyone interested in this, I have successfully reimplemented Jack Kelly's rectangles architecture using Keras and made it available here: https://github.com/maechler/nnilm

**Comment by JackKelly** [comment]
Great work!  Thanks!
